monkey/monkey_island/cc/repository/utils.py

```diff
@@ -33,3 +35,34 @@ def initialize_machine_repository(machine_repository: IMachineRepository):
             hostname=gethostname(),
         )
         machine_repository.upsert_machine(machine)
+
+
+DOT_REPLACEMENT = ",,,"
```

Comment:
We should consider something more unique than 3 commas. If the data is in CSV format, it's very likely there will be 3 commas in a row (empty fields) and the data won't encode.

We can use a string like "[MONGO_DOT_REPLACEMENT]". It's less likely you'll find that string in any valid data. We could add other stuff to make it even less likely. "[INFECTION_MONKEY_MONGO_DOT_REPLACEMENT]".

You get the idea.

Reply:
I wanted to leave it consistent with `_encode_dot_char_before_mongo_insert`. We can make it more unique, but it should be short in order to for keys to still be readable (if there's an IP address where each dot is replaced with "[INFECTION_MONKEY_MONGO_DOT_REPLACEMENT]" you'd need to do quire a bit of scrolling). I'd suggest something like `[_D0T_]`

Reply:
`[D0T]` sounds good to me.